toolkit/tools/validatechroot/validatechroot.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+	"microsoft.com/pkggen/internal/exe"
+	"microsoft.com/pkggen/internal/file"
+	"microsoft.com/pkggen/internal/logger"
+	"microsoft.com/pkggen/internal/safechroot"
+	"microsoft.com/pkggen/internal/shell"
+)
+
+var (
+	app = kingpin.New("validatechroot", "A tool to validate that the worker chroot is well configured and all dependencies are satisfied.")
+
+	toolchainRpmsDir = app.Flag("rpm-dir", "Directory that contains already built toolchain RPMs. Should contain top level directories for architecture.").Required().ExistingDir()
+	tmpDir           = app.Flag("tmp-dir", "Temporary chroot directory.").String()
+
+	workerTar      = app.Flag("worker-chroot", "Full path to worker_chroot.tar.gz").Required().ExistingFile()
+	workerManifest = app.Flag("worker-manifest", "Full path to the worker manifest file").Required().ExistingFile()
+
+	logFile  = exe.LogFileFlag(app)
+	logLevel = exe.LogLevelFlag(app)
+
+	leaveChrootFilesOnDisk = false
```

Comment:
This should be a `const` and defined as close to the scope of it's usage as possible